lib/versioning/Version.ts

```diff
@@ -190,6 +262,16 @@ export class Version {
         return this;
     }
 
+    /**
+     * Mark that the null version has been put with a Cloudserver handling null keys (i.e. supporting S3C-7352)
+     * 
+     * @return - the updated version
+     */
+    setNull2Version() {
+        this.version.isNull2 = true;
```

Comment:
I think we should also set `isNull` as we should never have `isNull2` without `isNull`. Even if we use this function only in the case `isNull` is already set, I think it doesn't hurt.